tests/dummy/app/components/exclaim-components/each/component.js

```diff
@@ -24,6 +24,10 @@ export const PROPERTIES = [
   }
 ];
 
+export const COMPONENT_META = {
+  boundKey: SHORTHAND_PROPERTY,
```

Comment:
I'm not sure- on one hand you can bind values to that key, but only in one way. I'll remove for now and only keep them in the explicit input components.

Reply:
That makes me think maybe we landed on the wrong name for this — the whole use case we're constructing it for is so that the wrapper can monitor the data the component writes, right? Maybe `writesKey` (or `writesKeys`?) captures that better?

Reply:
yeah I like that name, I'll change the existing ones